lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -696,7 +696,16 @@ void applyTextStyleToElement({
   final html.CssStyleDeclaration cssStyle = element.style;
 
   final ui.Color? color = style.foreground?.color ?? style.color;
-  if (color != null) {
+  if (style.foreground?.style == ui.PaintingStyle.stroke) {
```

Comment:
When comparing the outputs of the Bitmap Canvas and the DOM implementation, we have found, that we need to set the background color of the text to transparent to achieve the same effect as in the Bitmap Canvas and the Skia Engine that only the text stroke is painted.